src/Wechaty.Getting.Start/appsettings.json

```diff
@@ -7,5 +7,5 @@
     }
   },
   "WECHATY_PUPPET_SERVICE_TOKEN": "",
-  "WECHATY_PUPPET_SERVICE_Endpoint": ""
+  "WECHATY_PUPPET_SERVICE_Endpoint": "https://localhost:9002"
```

Comment:
I think this variable name should be renamed to `WECHATY_PUPPET_SERVICE_ENDPOINT` to keep its naming style consistent with the Wechaty ecosystem?